.github/workflows/build.yml

```diff
@@ -49,27 +49,17 @@ jobs:
          echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
          ./cmp_tags.sh
 
-     - name: Cache BuildX (BuildKit)
-       uses: actions/cache@v2
-       with:
-         path: /tmp/.buildx-cache
-         # Key should be unique for each run because if not we get an "exact cache hit" that will result not saving cache
-         key: pyinstaller-buildx-cache-${{ github.sha }}
-         restore-keys: |
-          pyinstaller-buildx-cache-
+     - name: Cache Docker Layers
+       uses: satackey/action-docker-layer-caching@v0.0.11
+       continue-on-error: true
 
-     - name: Prepare buildx
-       run: docker buildx create --name mybuilder --use
-      # When using DOCKER_BUILDKIT=1 docker is using moby-buildkit which uses other cache mechanism, we use
-      # `docker buildx build` cmd which is equivalent to DOCKER_BUILDKIT=1 and passing cache specific options.
-      # Another trick here is to set src and dst cache in different directories because we don't want the GH cache
-      # to grow very fast.
-      # BuildKit started working on github cache backend (https://github.com/moby/buildkit/pull/1974), need to migrate
-      # there when it becomes available.
      - name: Run Pyinstaller
        run: |
-        docker buildx build -t gprofiler-executable-builder --cache-from=type=local,src=/tmp/.buildx-cache \
-         --cache-to=type=local,dest=/tmp/.buildx-cache-new -f - --output type=local,dest=dist/ . < pyi.Dockerfile
+        docker build -t gprofiler-executable-builder -f - . < pyi.Dockerfile
```

Comment:
Explain in comment why this and not buildkit

Reply:
done